Helios/Interfaces/Profile/ProfileInterface.cs

```diff
@@ -243,21 +372,54 @@ private void CheckResetMonitors(IList<StatusReportItem> newReport)
                         Recommendation = "Perform Reset Monitors from the Profile menu",
                         Severity = StatusReportItem.SeverityCode.Error,
                         Link = StatusReportItem.ProfileEditor
+                        // TODO handle reporting tablet orientated incorrectly for the profile - ie profile width = screen height and profile height = screen width - the action would be to rotate the device rather than perform a monitor reset
                     });
                 }
             }
         }
 
         /// <summary>
-        /// check configuration related to launching or stopping executables
+        /// check configuration related to launching executables
         /// </summary>
         /// <param name="newReport"></param>
         private void CheckExecutablePaths(IList<StatusReportItem> newReport)
         {
-            // TODO if any launch or kill actions are configured, check paths and access
-            // TODO if incorrect, report warning or error (to stop launch) in StatusReportItem
-            // TODO if correct, report detail also in StatusReportItem
-            _ = newReport;
+            foreach(HeliosBinding _binding in OutputBindings)
+            {
+                if(_binding.Action.ActionID == "launch application")
```

Comment:
switch on binding.Action.ActionID and log in default case that you found an unimplemented / unexpected command

C# does switch/case on strings

Reply:
btw probably fine to intentionally throw Null exception (i.e. do nothing) if binding.Action is null.  I think that would be a structural flaw that SHOULD come back as an exception stack trace from the field.   So I would leave it unguarded like you have it.

Reply:
I think if we wish to validate all of the actions, then this code needs to go elsewhere.  Since CheckExecutablePaths has a very particular purpose, I propose leaving this code as is.

Reply:
I just mean if then else if is inferior to switch / case for readability and has less chance of being broken by a careless edit later.   if these are not all the actions, then of course don't log in the default :)

but if then else if on the same variable is still a switch/case in my book.

Reply:
All for readability